apps/happy-blocks/block-library/search-card/index.php

```diff
@@ -7,10 +7,20 @@
  * @package happy-blocks
  */
 
+/**
+ * Load functions from the h4 theme, we're using localized_wpcom_url here.
+ */
+require_once WP_CONTENT_DIR . '/themes/h4/landing/marketing/pages/_common/lib/functions.php';
```

Comment:
I'm surprised we need to import this. https://opengrok.a8c.com/source/xref/wpcom/.config/locales.php?r=e227e470&mo=5782&fi=143#143